projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/execution/JobExecutionService.java

```diff
@@ -196,7 +198,7 @@ public void cancelDataJobExecution(String teamName, String jobName, String execu
 
       return dataJobExecutions
             .stream()
-            .map(dataJobExecution -> ToApiModelConverter.jobExecutionToConvert(dataJobExecution))
+            .map(dataJobExecution -> ToApiModelConverter.jobExecutionToConvert(dataJobExecution, jobExecutionLogsUrlBuilder.build(dataJobExecution)))
```

Comment:
You probably need to update `getLastExecution` as well . 

You may consider some common helper method as well.

Reply:
Do you mean JobExecutionService.getLastExecution()?

Reply:
Common method that wraps `jobExecutionLogsUrlBuilder.build(dataJobExecution)` or `ToApiModelConverter.jobExecutionToConvert(dataJobExecution, jobExecutionLogsUrlBuilder.build(dataJobExecution))`?

Reply:
I was thinking about such a method (`ToApiModelConverter.jobExecutionToConvert(dataJobExecution, jobExecutionLogsUrlBuilder.build(dataJobExecution))`) but I didn't figure out where to put it. Do you have any suggestion?

Reply:
> Do you mean JobExecutionService.getLastExecution()? If yes it uses DB entity not the model.

Oh right, They both have the same name and I glossed over the package name.

Reply:
> I was thinking about such a method (ToApiModelConverter.jobExecutionToConvert(dataJobExecution, jobExecutionLogsUrlBuilder.build(dataJobExecution))) but I didn't figure out where to put it. Do you have any suggestion?

It's a nit really. I meant local private method